### PR TITLE
Vector tests 

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

**Comment by conorbros** [comment]
Lets get this in, I'll open another PR for the part with encoding I'm trying to fix.

File: tests/test-configs/cassandra/tls/certs/localhost.crt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-----BEGIN CERTIFICATE-----
```
Comment (rukai) [inline tests/test-configs/cassandra/tls/certs/localhost.crt:1]: accidental checkin

**Review by benbromhead — APPROVED** [review]

**Comment by rukai** [comment]
This was marked as "ready to review" but the PR still depends on your personal cdrs-tokio fork.
I think this needs to remain as a draft until the cdrs-tokio changes land upstream.

We can be a bit more lenient with dependencies only used for tests, but if shotover itself uses git deps then we will be unable to publish new shotover releases to crates.io
And the cdrs-tokio upstreaming PR is still a draft.

**Comment by conorbros** [comment]
Thanks. I've updated https://github.com/krojew/cdrs-tokio/pull/179 and will progress this PR once it's merged.

**Comment by rukai** [comment]
Cassandra 5 vector support was added by https://github.com/shotover/shotover-proxy/pull/1676

Thanks for your early investigations and work on vector support.

**Pull request closed without merging**